### PR TITLE
Feature/3246/remove ngx bootstrap hosted workflows

### PR DESCRIPTION
**Pull request by ByteMap**
Removed ngx-bootstrap tab from hosted workflows

dockstore/dockstore#3246

Migrated from using ngx-bootstrap tabs to Material Tabs in the
hosted workflows page.

Imported BrowserAnimationsModule to the spec file in order for tests
to pass

Removed an unused ngx-bootstrap module and method from home.component.ts

Adjusted hostedWorkflows cypress tests to work with the new Material Tabs

### CODE REVIEW

**Comment by coverbeck** [comment]
Could you please attach a screenshot(s) of how it looks now?

File: src/app/home-page/home-logged-out/home.component.ts [hunk 1]
```diff
@@ -88,10 +87,6 @@ export class HomeComponent extends Base implements OnInit, AfterViewInit {
     this.homePageService.goToSearch(searchValue);
   }
 
-  onSelect(data: TabDirective): void {
```
Comment (coverbeck) [inline src/app/home-page/home-logged-out/home.component.ts]: To be clear, this is unrelated to the crux of your PR, right? Is it just a dangling reference to ngx-bootstrap that wasn't being used?

Comment (ByteMap) [reply src/app/home-page/home-logged-out/home.component.ts]: Yes it is just a dangling unused reference to ngx-bootstrap. I figured that since it was a very easy fix, I would just include it into this PR.

**Comment by ByteMap** [comment]
Display of using Material Tabs instead of ngx-bootstrap tabs for hostedWorkflows. 
Screen Shot 2020-03-02 at 3 32 12 PM
Screen Shot 2020-03-02 at 3 32 23 PM
